tensorboard/webapp/plugins/npmi/views/main/main_component.scss

```diff
@@ -22,7 +22,8 @@ limitations under the License.
 
 .content {
   display: flex;
-  height: 100%;
+  flex: 1;
+  min-height: 0px;
```

Comment:
Is the `min-height: 0` necessary?  I wouldn't be too surprised if it is, but just checking.

Reply:
Yes, this prevents the container from growing larger than it should. CSS is just so intuitive... :D